spec/meta_inspector/redirections_spec.rb

```diff
@@ -47,6 +47,25 @@
         expect(page.url).to eq("http://blogs.clarionledger.com/dechols/2014/03/24/digital-medicine/?nclick_check=1")
       end
     end
+
+    context "when there is a callback to be ran between redirects that blocks redirections to private IP addresses" do
+      it "raises an exception" do
+        stub_request(:get, "https://www.facebook.com/")
+          .to_return(:status => 302,
+                     :headers => { "Location" => "http://10.0.0.0/" })
+
+        redirect_options = {
+          callback: proc do |_previous_response, next_request|
+            ip_address = Resolv.getaddress(next_request.url.host)
```

Comment:
I don't have the `Resolv` gem installed, and I don't think it should be added as a dependency to MetaInspector as it's only used for a spec.

How about simplifying this spec? It would be enough to have the callback raise an exception is the redirection goes to 10.0.0.0 for example.

Maybe Resolv can be mentioned in the Readme as an example / suggestion on controlling private address redirections.

Reply:
On a second thought, we could add the `Revolv` gem only as a development dependency, in `meta_inspector.gemspec`

```
gem.add_development_dependency 'resolv', '~> 0.2.2'
```

and then at the top of `redirections_spec.rb`:

```ruby
require 'resolv'
```